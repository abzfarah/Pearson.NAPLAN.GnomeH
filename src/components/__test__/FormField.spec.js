// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react/lib/ReactTestRenderer';

import FormField from '../FormField';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('FormField', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <FormField label="Item 1" htmlFor="item1">
        <input id="item1" type="text" />
      </FormField>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
