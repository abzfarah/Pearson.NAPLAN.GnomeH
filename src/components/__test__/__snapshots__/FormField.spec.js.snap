exports[`FormField has correct default options 1`] = `
<div
  className="grommetux-form-field grommetux-form-field--text grommetux-form-field--size-medium"
  onClick={[Function bound _onClick]}>
  <label
    className="grommetux-form-field__label"
    htmlFor="item1">
    Item 1
  </label>
  <span
    className="grommetux-form-field__contents">
    <input
      id="item1"
      type="text" />
  </span>
</div>
`;
